app/src/main/res/layout/activity_link_device.xml

```diff
@@ -9,38 +9,38 @@
     <androidx.viewpager.widget.ViewPager
         android:id="@+id/viewPager"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" >
+        android:layout_height="match_parent">
 
         <com.google.android.material.tabs.TabLayout
-            style="@style/Widget.Session.TabLayout"
             android:id="@+id/tabLayout"
+            style="@style/Widget.Session.TabLayout"
             android:layout_width="match_parent"
             android:layout_height="@dimen/tab_bar_height" />
 
-    </androidx.viewpager.widget.ViewPager>
```

Comment:
Looks like there is an unintentional layout change here (the `FrameLayout` is now within the `ViewPager` - this is likely what is causing the onboarding loading issue)

Reply:
Ok I've added it back in... not sure why that was removed

Reply:
Might have been a weird auto-formatting thing or caused by moving something around not too sure

Reply:
Just added another comment - it looks like the line was shifted down rather than removed so we now have two `</androidx.viewpager.widget.ViewPager>` tags (should just need to remove the latter one)

Might also be good to double check to make sure this does actually resolve the loading issue you were seeing @burtonemily